code_examples/core_features/web3names/03_query_name_credentials.ts

```diff
@@ -78,7 +78,7 @@ export async function queryPublishedCredentials(
   // The case where the result is not a JSON should be properly handled in production settings.
   const credentialCollection: CredentialEntry[] = await fetch(
     firstCredentialCollectionEndpointUrl
-  ).then((response) => response.json())
+  ).then((response) => response.json().then((r) => r as CredentialEntry[]))
```

Comment:
The extra then just for the sake of the type system is unnecessary I think. If you really want/must add extra type annotation, use
```suggestion
  ).then((response) => (response.json() as Promise<CredentialEntry[]>))
```